### PR TITLE
feat: 在树状模式下，增加默认展开层级的配置

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [x] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/common/interface/basic.ts [hunk 1]
```diff
@@ -194,6 +194,8 @@ export interface Style {
   treeRowsWidth?: number;
   // 树状分层模式下的全局收起展开属性，对应角头收起展开按钮
   hierarchyCollapse?: boolean;
+  // 树状分层模式下，行头默认展开到第几层
+  expandToRowLevel?: number;
```
Comment (wjgogogo) [inline packages/s2-core/src/common/interface/basic.ts:198]: 命名看看大家的意见

Comment (lijinke666) [reply packages/s2-core/src/common/interface/basic.ts:198]: rowExpandDepth ?

File: s2-site/docs/common/style.zh.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ object **必选**,_default：null_ 功能描述：样式设置
 | showTreeLeafNodeAlignDot | `boolean` |  |  false  | 树状模式下叶子节点是否显示层级占位点 |
 | treeRowsWidth | `number` |  |  120  | 树状模式行单元格宽度 （优先级大于 `rowCfg.width` 和 `rowCfg.treeRowsWidth （已废弃）`) |
 | hierarchyCollapse | `boolean` |  |   `false` | 在树状结构模式下行头是否默认展开。 |
+| expandToRowLevel | `number` |  |    | 在树状结构模式下行头默认展开展开的层级（层级从 0 开始）。 |
```
Comment (lijinke666) [inline s2-site/docs/common/style.zh.md:16]: 加个 demo 哇?

**Review by lijinke666 — DISMISSED** [review]

**Review by lijinke666 — APPROVED** [review]

File: packages/s2-core/src/common/interface/basic.ts [hunk 3]
```diff
@@ -194,6 +194,8 @@ export interface Style {
   treeRowsWidth?: number;
   // 树状分层模式下的全局收起展开属性，对应角头收起展开按钮
   hierarchyCollapse?: boolean;
+  // 树状分层模式下，行头默认展开到第几层
+  rowExpandDepth?: number;
```
Comment (xingwanying) [inline packages/s2-core/src/common/interface/basic.ts:198]: rowExpandLevel？

**Review by stone-lyl — APPROVED** [review]

File: packages/s2-core/src/common/interface/basic.ts (hunk 3 above)
Comment (wjgogogo) [reply packages/s2-core/src/common/interface/basic.ts:198]: 这两个之间差别不大吧

**Pull request merged**